### ISSUE TITLE
Place exported glyph origin at left side of baseline

### ISSUE THREAD

**Issue by fpdotmonkey** [coldtype/st2#19]
I'm trying to create an animation using some generated text similar to the Universal Studios logo animation at the start of their movies.  However, I'm finding that the placement of the origin makes this difficult or impossible.  Here is the procedure I'm attempting.

1. Create the text I'd like to animate
2. Export Glyphs for animation
3. Add constraints to the exported glyphs to maintain the inter-character distance (ISSUE: the glyph origins don't line up along the baseline axis, making creating an effective constraint difficult/impossible)
4. Add a Follow Path constraint for my particular application
5. Create the animation

The issue in step 3 can be worked around by wrapping each glyph in a parent object whose origin is on the baseline.

It would be better if the origin of the glyph were normally placed at a typographically important coordinate, e.g. the typographical left side on the baseline.

**Comment by stenson** [comment]
makes sense! been meaning to add something like this — think I've got something working, here are some builds depending on OS:

- coldtype.xyz/st2/releases/ST2-v0-13_mac_intel.zip
- coldtype.xyz/st2/releases/ST2-v0-13_mac_silicon.zip
- coldtype.xyz/st2/releases/ST2-v0-13_win.zip

<img width="779" alt="image" src="https://github.com/coldtype/st2/assets/89770/27313e1d-9459-4e1f-8ad6-d15cdd8921e7">

Think you'd want "Southwest" and typographic x & y turned on (as pictured in the draft UI here)

Let me know if that's what you have in mind

**Comment by fpdotmonkey** [comment]
That's exactly what I had in mind, yeah!  Thanks for implementing it so quickly.
